docs/installation_c_plus_plus.md

```diff
@@ -0,0 +1,129 @@
+# C++ Installation
+
+!!! note
+      This version of the documentation is for installation of Xailient SDK using wheel file (.whl). 
```

Comment:
These are C++ installation instructions. But this talks about a (python) .whl file, not a .tar.gz file that contains the C++ SDK.